articles/variables-in-js/index.md

```diff
@@ -0,0 +1,129 @@
+# Understanding Variables In Javascript
+
+## Introduction
+Variables are a way for programmers to name a value to reuse it, update it, or keep track of it.
+In this article, we will go through naming variables, declaring them, and initializing them.
+We are also going to touch on the scope of variables to ensure that we understand variables fully.
+
+## Prerequisites
+To follow along with this tutorial, you need a web browser. You can get Google Chrome [here](https://www.google.com/chrome/)
+
+### Accessing the JavaScript Console
+We will use the JavaScript console to execute the JavaScript code in this tutorial. To open it in Google Chrome, press `Ctrl + Shift + J` on Windows and Linux and `Cmd + Option + J` in Mac OS.
+
+### Understanding the scope of variables
+Scope determines the accessibility of variables, objects, and functions from different parts of your code.
+Javascript has two types of scopes, which are:
+1. Global scope
+2. Local scope
+
+**1. Global scope**
+
+When a variable is *globally scoped*, it means it is available anywhere in your program. 
+Declaring a variable outside a function leads to it being ade to the global scope automatically.
+Global scope uses `var` key word to declare variables.
```

Comment:
Not necessarily, you can have a `const` and `let` global variable too.

Reply:
Once you declare any variable using any keyword outside a block or a function, it's global.

Reply:
I think you can remove this line.